components/builder-minio/habitat/config_install/minio-migration.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+function _hab_exec() {
+  pkg=$1
+  shift
+  hab pkg exec "$pkg" -- "$@"
+}
+
+function _aws() {
+  _hab_exec 'core/aws-cli' aws --endpoint-url "{{cfg.env.MINIO_ENDPOINT}}" "$@"
+}
+
+function _curl() {
+  _hab_exec 'core/curl' curl "$@"
+}
+
+function _jq() {
+  _hab_exec 'core/jq-static' jq "$@"
+}
+
```

Comment:
As long as aws, curl, and jq are runtime deps, you don't need `hab pkg exec`. You can call them directly because they will be on the path.

Reply:
Removed wrappers and called directly.